markdown/exploring-solace-native-apis/exploring-solace-native-apis.md

```diff
@@ -0,0 +1,601 @@
+author: Tom Fairbairn
+summary: An exploration of the basics of getting started with Solace's Native APIs
+id: exploring-solace-native-apis
+tags: workshop
+categories: Java
+environments: Web
+status: Draft
+feedback link: A link where users can go to provide feedback (e.g. the git repo or issue page)
```

Comment:
Add a feedback link (https://github.com/SolaceSamples for example) or keep it empty here